staging/src/k8s.io/api/apps/v1/types.go

```diff
@@ -831,7 +831,7 @@ type ReplicaSetSpec struct {
 
 // ReplicaSetStatus represents the current status of a ReplicaSet.
 type ReplicaSetStatus struct {
-	// Replicas is the most recently oberved number of replicas.
+	// Replicas is the most recently observed number of replicas.
```

Comment:
it looks like the same issue exists in a few other versions of the types.go files... mind fixing them all in this PR?

Reply:
Fixed.